administrator/components/com_users/src/Model/UserModel.php

```diff
@@ -1034,19 +1034,22 @@ public function getOtpConfig($userId = null)
 		if (strpos($config, '{') === false)
 		{
 			$openssl         = new Aes($key, 256);
-			$mcrypt          = new Aes($key, 256, 'cbc', null, 'mcrypt');
+
+			// Deal with legacy mcrypt encrypted data, and convert it to openssl encrypted data.
+			$mcrypt          = new Aes($key, 256, 'cbc', 'mcrypt');
```

Comment:
```suggestion
			$mcrypt = new Aes($key, 256, 'cbc', 'mcrypt');
```
If not directly below another assignment anymore, it doesn't need to align the "=".

Reply:
yup I was just adding more comments - sorry for the delay